pkg/plugins/resources/dockerimage/main.go

```diff
@@ -7,30 +7,36 @@ import (
 	"github.com/mitchellh/mapstructure"
 	"github.com/updatecli/updatecli/pkg/plugins/utils/docker/dockerimage"
 	"github.com/updatecli/updatecli/pkg/plugins/utils/docker/dockerregistry"
+	"github.com/updatecli/updatecli/pkg/plugins/utils/version"
 )
 
 // Spec defines a specification for a "dockerimage" resource
 // parsed from an updatecli manifest file
 type Spec struct {
-	// Architecture specifies the container image architecture such as `amd64`
+	// [S][C][T]Architecture specifies the container image architecture such as `amd64`
```

Comment:
Shouldn't a space be added between the last `]` and the first comment word on these lines?

Reply:
Good point